src/Idris/Elab/Clause.hs

```diff
@@ -706,8 +704,7 @@ elabClause info opts (cnum, PClause fc fname lhs_in_as withs rhs_in_as wherebloc
 
         mapM_ (\(n, _) -> addIBC (IBCDef n)) def''
 
-        when (not (null def')) $ do
-           mapM_ defer_totcheck (map (\x -> (fc, fst x)) def'')
+        when (not (null def')) $ mapM_ defer_totcheck (map (\x -> (fc, fst x)) def'')
```

Comment:
It's good to get rid of "do" here, but I still think the part after the "$" should be indented on the next line, because that looks more like the imperative control structure that "when" encodes.